src/main/java/com/greenfoxacademy/moviedatabaseapi/service/MyUserDetailsService.java

```diff
@@ -0,0 +1,25 @@
+package com.greenfoxacademy.moviedatabaseapi.service;
+
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+
+@Service
+public class MyUserDetailsService implements UserDetailsService {
+
+    private final PasswordEncoder passwordEncoder;
+
+    public MyUserDetailsService(PasswordEncoder passwordEncoder) {
+        this.passwordEncoder = passwordEncoder;
+    }
+
+    @Override
+    public UserDetails loadUserByUsername(String userName) throws UsernameNotFoundException {
+        return new User("Joe", passwordEncoder.encode("password"), new ArrayList<>());
```

Comment:
not sure who "Joe" is... seems to be sth is not finished
userName param is not used